R/ilpd.R

```diff
@@ -0,0 +1,29 @@
+#' @title Indian Liver Patient Dataset
+#'
+#' @name ilpd
+#' @aliases mlr_tasks_ilpd
+#'
+#' @description
+#' Classification data to predict whether or not a person is a liver patient.
+#' Obtained using the \CRANpkg{OpenML} package. Contains 538 observations and 10
```

Comment:
mlr3oml?

Reply:
Ah yes. Also fixed this for moneyball and optdigits.